docs/administration-guide/gaffer-stores/federated-store.md

```diff
@@ -2,4 +2,406 @@
 
 The Federated Store is a Gaffer store which forwards operations to a collection of sub-graphs and returns a single response as though a single graph were queried.
 
-For more information on the federated store, see [the v1 federated store walkthrough](https://gchq.github.io/gaffer-doc/v1docs/getting-started/developer-guide/federatedstore.html) (to be updated and added to this documentation in future).
\ No newline at end of file
+## Introduction
+
+The Federated Store facilitates querying of multiple Gaffer graphs through a single endpoint/instance, and the federation of queries across many graphs at once - with results automatically merged.
+Unlike other Stores, the Federated Store doesn't use a schema of its own. This is because it represents a collection of other graphs, these can be referred to as sub-graphs.
+The Federated Store 'graph' represents a way to query these sub-graphs and is not an actual graph.
+
+Federated Store sub-graphs can be any kind of Gaffer graph, including further federated graphs. It's important to remember that it doesn't store data on its own.
+There are [special Gaffer Operations](#performing-operations) for adding and removing graphs to/from the collection. It's also possible to include access restrictions on sub-graphs.
+When adding a new sub-graph, the schema and store properties for this new graph are supplied to the Federated Store directly.
+
+The Federated Store can be used as a single entry point to a set of multiple different Gaffer graphs, using different store types and across different instances/hosts (via Proxy Stores).
+
+## Create a Federated Store
+
+To create a Federated Store, you need to initialise the store/graph with a `graphId` and a properties file.
+See [the Graph page for detailed info](../../development-guide/project-structure/components/graph.md#creating-a-graph) on this.
+
+Optionally you can add a [Graph Library](../../development-guide/project-structure/components/graph.md#graph-configuration) to store Schemas and Store Properties associated with the Federated Store sub-graphs.
+You can also add a [Cache Service](store-guide.md#caches) within the Store Properties, for example this will be required if you want to use Named Operations.
+
+### Using the REST API
+
+When using the REST API, configuration is done with the files `graphId.json` and `store.properties`. Minimal examples are shown below.
+
+```json title="graphId.json"
```

Comment:
Although it would still work like this, is this meant to be `graphConfig.json`?

Reply:
This was indeed a typo